polars/lazy/dataframe.ts

```diff
@@ -12,6 +12,7 @@ import {
 import { _LazyGroupBy, LazyGroupBy } from "./groupby";
 import { Deserialize, GroupByOps, Serialize } from "../shared_traits";
 import { LazyOptions, LazyJoinOptions } from "../types";
+import { Series } from "@polars/series";
```

Comment:
A heads up that this absolute import is causing my downstream CI checks to fail on the upgrade PR for the latest version of nodejs-polars because typescript transpilation is failing:

```
Cannot find module '@polars/series' or its corresponding type declarations.
```

I can probably tweak my tsconfig to work around this, but thought it was worth flagging since it's inconsistent with the rest of the imports. Maybe something you want to codify for the project.

Reply:
I'll look into it on monday

Reply:
https://github.com/pola-rs/nodejs-polars/pull/76